app/src/full/java/io/homeassistant/companion/android/notifications/MessagingService.kt

```diff
@@ -206,8 +209,13 @@ class MessagingService : FirebaseMessagingService() {
 
         val notificationBuilder = NotificationCompat.Builder(this, channelId)
             .setSmallIcon(R.drawable.ic_stat_ic_notification)
-            .setSound(RingtoneManager.getDefaultUri(RingtoneManager.TYPE_NOTIFICATION))
 
+        if (data["channel"] == "alarm_stream") {
+            notificationBuilder.setCategory(Notification.CATEGORY_ALARM)
+            notificationBuilder.setSound(RingtoneManager.getDefaultUri(RingtoneManager.TYPE_ALARM), AudioManager.STREAM_ALARM)
+        } else {
+            notificationBuilder.setSound(RingtoneManager.getDefaultUri(RingtoneManager.TYPE_NOTIFICATION))
+        }
```

Comment:
We should break the sound handling into a method like the rest of the items.  I think the category can go in there for now as well.